internal/bcdb/transaction_processor.go

```diff
@@ -304,13 +366,16 @@ func (t *transactionProcessor) SubmitTransaction(tx interface{}, timeout time.Du
 		}
 	}
 	t.logger.Debug("transaction is enqueued for re-ordering")
+	t.metrics.QueueSize("tx", t.txQueue.Size())
+	t.metrics.QueueSize("pending", t.pendingTxs.Size())
```

Comment:
Lets use these two metrics to test the metrics endpoint server from the out side. once we lay the infrastructure for it, and test it, we can easily add metrics to additional units following the same pattern (i.e. handlers, etc).